replicated-security/pion-1/v1.1.0_upgrade.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Neutron v1.1.0 Upgrade
+order: 2
+---
+<!-- markdown-link-check-disable -->
+
+# Neutron v1.1.0 Upgrade, Instructions
+
+* Chain upgrade point: TODO: DETERMINE AND ADD HEIGHT AND DATE
+* Go version: v1.20
+* Release: TODO: ADD A LINK TO THE RELEASE
+
+This document describes the steps for validators and full node operators, to upgrade successfully to the Neutron v1.1.0 release. For more details on the release, please see the [release notes](TODO:ADD A LINK TO THE RELEASE).
+
+## On-chain governance proposal attains consensus
+
+[Proposal N](TODO: ADD A LINK TO THE PROPOSAL) is the reference on-chain governance proposal for this upgrade. Neither core developers nor core funding entities control the governance, and this governance proposal has passed in a _fully decentralized_ way.
+
+## Upgrade date
+
+The upgrade will take place at a block height of `TODO: ADD THE DETERMINED BLOCK HEIGHT`. The date/time of the upgrade is subject to change as blocks are not generated at a constant interval. You can stay up-to-date using this [live countdown](https://www.mintscan.io/neutron/blocks/TODO: ADD THE DETERMINED BLOCK HEIGHT) page.
+
+## Chain-id will remain the same
+
+The chain-id of the network will remain the same, `neutron-1`. This is because an in-place migration of state will take place, i.e., this upgrade does not export any state.
+
+### System requirement
+
+32GB RAM is recommended to ensure a smooth upgrade.
```

Comment:
64gb is recommended.
During local testing on a chain fork, binary consumes up to 55gb.